src/app/core/core.module.ts

```diff
@@ -109,3 +108,38 @@ export { User } from './auth/user.model';
 export { Role } from './auth/role.model';
 export { ConfigService } from './config.service';
 export { NavbarTopics } from './site-navbar/navbar-topic.model';
+
+AdminTopics.registerTopic({
```

Comment:
Admin topic registration had to be moved out of the lazy loaded module and into the core app.  Was debating between putting here or above in the module's constructor.  Open to suggestions if anyone has a better idea.